content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -16,17 +16,276 @@ further_reading:
+By default, the Datadog tracing libraries read and write distributed tracing headers using both the **Datadog** format and the [**W3C Trace Context**][2] formats. When reading incoming headers, the language SDK's will giver higher precedence to the Datadog format, whereas Envoy and NGINX proxies will give higher precedence to the W3C Trace Context format.
 
-{{< partial name="apm/apm-context-propagation" >}}
+## Configuration
 
+If you need to customize the trace context propagation configuration, there are several environment variables you can use to configure the formats that are used for reading and writing distributed tracing headers. To enable a specific propagator, make sure to use the corresponding configuration value for the tracing library, as outlined in the **Language support** section.
 
-<br>
+<div class="alert alert-info">
+If multiple propagators are enabled, the extraction attempt is done in the specified order, and the first valid trace context is used to continue the distributed trace. If additional valid trace contexts are found, the tracing information will be recorded as individual span links.</div>
+
+### Recommended Datadog configuration
+
+`DD_TRACE_PROPAGATION_STYLE`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This may be overridden by the extract-specific or inject-specific configurations. <br>
+**Default:** `datadog,tracecontext` for language SDK's, `tracecontext,datadog` for Envoy and NGINX procies
+
+### Recommended OpenTelemetry configuration
+
+`OTEL_PROPAGATORS`
+: Specifies propagators (in a comma-separated list) to be used for trace context propagation. This configuration takes the lowest precedence and will be ignored if any other Datadog propagation environment variable is set.
+
+### Advanced configuration
+
+In the majority of scenarios, you will want to both send and receive trace context headers using the same format. However, if your service needs to accept trace context headers in one format and send them in another format, you can customize them with the following configurations.
+
+`DD_TRACE_PROPAGATION_STYLE_EXTRACT`
+: Specifies propagators (in a comma-separated list) to be used only for trace context extraction. This configuration takes the highest precedence over all other configurations for configuring the extraction propagators.
+
+`DD_TRACE_PROPAGATION_STYLE_INJECT`
+: Specifies propagators (in a comma-separated list) to be used only for trace context injection. This configuration takes the highest precedence over all other configurations for configuring the injection propagators.
+
+## Language support
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+### Supported propagators
+The Datadog Java SDK maintains several propagators for passing trace context information in different formats:
+
+| Propagator             | Configuration Value |
```

Comment:
```suggestion
| Format             | Configuration Value |
```

Reply:
Fixed in ca792a35e1ce6b68fe8d312c91884e25eb1d6e46